BackofficeBundle/Controller/NodeController.php

```diff
@@ -54,19 +54,21 @@ public function formAction(Request $request, $id)
      * @Config\Route("/node/new/{parentId}", name="open_orchestra_backoffice_node_new")
      * @Config\Method({"GET", "POST"})
      *
-     * @Config\Security("is_granted('ROLE_ACCESS_CREATE_NODE')")
-     *
      * @return Response
      */
     public function newAction(Request $request, $parentId)
     {
+        $parentNode = $this->get('open_orchestra_model.repository.node')->findOneByNodeId($parentId);
+
+        $creationRole = $parentNode->getNodeType() === NodeInterface::TYPE_TRANSVERSE? GeneralNodesPanelStrategy::ROLE_ACCESS_GENERAL_NODE:TreeNodesPanelStrategy::ROLE_ACCESS_CREATE_NODE;
+        $this->denyAccessUnlessGranted($creationRole, $parentNode);
```

Comment:
Cause we have to check the parent node creation role. The 'role_access_create_node' allows users to create a node inside the actual node right?
We can't create a node and test after if we have the right to create it.